src/views/orb-view.ts

```diff
@@ -150,6 +168,8 @@ export class OrbView<N extends INodeBase, E extends IEdgeBase> implements IOrbVi
       .on('contextmenu', this.mouseRightClicked)
       .on('dblclick.zoom', this.mouseDoubleClicked);
 
+    document.addEventListener('keydown', this._handleKeyDown);
```

Comment:
Ok, this is a bit problematic, to listen for the key down events on the whole document. If there is a form next to the Orb view, orb will get each event and even react to +, -, array keys while the user writes keys in the input.

So, this should be fixed either by:

* Listening only for keys on the container - I don't think this is even possible
* Having a way to listen for click events in the container to make it "focused" so we can know if we want to react to document.keydown event (for example, check the `orb-view-map` and how it works - key events are working as expected there: if you click on a map, it becomes "focused" in the background, and keys are working. If you click on the header, keys are not working for the graph view out of the orb view, which is the expected behavior). 

If you have any other suggestions on how to approach this problem, feel free to share. Happy to discuss this.

Reply:
@parmar-abhinav do you have any info on this?

Reply:
Tried few approach i.e.
1. Instead of attaching the `keydown` event listener to the entire document. I tried attaching it to the specific container where we want to handle the key events. But not able to listen to `keydown` event using this approach.
```typescript
this._container.addEventListener('keydown', (event) => {})
```

2. Also tried listening to `focusin` and `focusout` event on the container. But it is also not working.
```typescript
private readonly isContainerFocused = false;

this._container.addEventListener('focusin', () => {
    this.isContainerFocused = true;
});

this._container.addEventListener('focusout', () => {
    this.isContainerFocused = false;
});

document.addEventListener('keydown', event => {
    if (this.isContainerFocused) {
        // Handle keydown events only when the container is focused
        this._handleKeyDown(event);
    }
});

```

Reply:
Hmm, thanks for sharing. I will check how the leaflet did it (it is used for orb-view-map), maybe we can replicate that behavior. I will share my findings here.